src/enochecker/utils.py

```diff
@@ -44,9 +44,10 @@ def assert_in(o1: Any, o2: Any, message: Optional[str] = None) -> None:
     :param message: An optional message that will be part of the error
     """
     if message is None:
-        message = "{} is not in {}".format(o1, o2)
+        message = "Did not receive expected response"
```

Comment:
Is this message all that useful though? I'd say we should rather just display no info if message is None

Reply:
For a team you mean? More useful than no response. Mumble could have a variety of reasons,  didn't receive the correct response is more precise imho